src/Test/QuickCheck/StateMachineModel/Example.hs

```diff
@@ -67,12 +67,12 @@ transitions m cmd _ = case cmd of
   Write ref i -> update m (unRef ref) i
   Inc   ref   -> update m (unRef ref) ((m ! ref) + 1)
 
-postconditions :: Model -> MemStep resp  -> resp -> Bool
+postconditions :: Model -> MemStep resp  -> resp -> Property
 postconditions m cmd resp = case cmd of
-  New         -> True
-  Read ref    -> m  ! ref == resp && m' == m
-  Write ref i -> m' ! ref == i
-  Inc   ref   -> m' ! ref == m ! ref + 1
+  New         -> property $ True
+  Read ref    -> m  ! ref === resp .&&. m' == m
+  Write ref i -> property $ m' ! ref == i
```

Comment:
couldn't this also be `m' ! ref === i`? I'm not sure about the combinators of `Property` but this looks inconsistent between `Read` and `Write`, `Inc`.